client/common/services/canvas.service.js

```diff
@@ -2,14 +2,29 @@
     "use strict";
     
     angular
-        .module("services", [])
-        .service("CanvasService", CanvasService);
+        .module("smitty.services", [])
```

Comment:
Forgot to put this in the commit message. I added the `smitty` namespace to your modules. You can change it to something else if you want. It's good to prefix your own modules with a custom namespace so that if you use another package where someone has a "services" module, they won't conflict.